src/main/frontend/model/samples/add-node.json

```diff
@@ -0,0 +1,43 @@
+{
+	"logLevel": "info",
+	"hops": [
+		{
+			"type": "declare",
+			"declarations": {
```

Comment:
IMHO, for all the scripts in this PR, try using parameters in place of global declarations. It’s what they are meant for.